dotnet-nunit/SauceLabs.Visual.Example/SauceDemo.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Remote;
+using OpenQA.Selenium.Support.UI;
+using SauceLabs.Visual.Models;
+using SauceLabs.Visual;
+
+namespace SauceLabs.Visual.Example;
+
+public class SauceDemo
+{
+    private RemoteWebDriver? Driver { get; set; }
+    private VisualClient? VisualClient { get; set; }
+
+    [OneTimeSetUp]
+    public async Task Setup()
+    {
+        var browserOptions = Utils.GetBrowserOptions();
+        var sauceOptions = Utils.GetSauceOptions();
+        browserOptions.AddAdditionalOption("sauce:options", sauceOptions);
+
+        var sauceUrl = Utils.GetOnDemandURL();
+        Driver = new RemoteWebDriver(sauceUrl, browserOptions);
+        Driver.ExecuteScript("sauce:job-name=NUnit C#/.Net Visual Session");
+
+        VisualClient = await VisualClient.Create(Driver, Region.UsWest1, new CreateBuildOptions()
+        {
+            Name = "My Visual Build",
+            Project = "csharp-project",
+            Branch = "csharp-branch"
+        });
+        VisualClient.CaptureDom = true;
+        TestContext.Progress.WriteLine($"BuildId: {VisualClient.Build.Url}");
+    }
+
+    [Test]
+    public async Task SauceDemo_CheckInventory()

```

Comment:
Are you following C# testing convention?
```suggestion
    public async Task SauceDemo_CheckInventory_ShouldAddBackpackToCart()

```